index.html

```diff
@@ -21,18 +23,198 @@
+          <p>Back End Dev</p>
+          <p>2015</p>
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li> 
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
+     <!-- 2nd project card -->
+    <article class="project-card"> 
+      <div class="project-img">
+        <img src="assets/Snapshoot-2.svg" alt="Screenshot of Project 2">
+      </div>
+      <div class="project-content">
+        <h2>Multi-Post Stories</h2>
+        <div class="project-info">
+          <h3>CANOPY</h3>
+          <p>Back End Dev</p>
+          <p>2015</p>
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li> 
+          <li><a href="#">CSS</a></li> 
+          <li><a href="#">JavaScript</a></li> 
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
+     <!-- 3rd project card -->
+    <article class="project-card"> 
+      <div class="project-img">
+        <img src="assets/Snapshoot-3.svg" alt="Screenshot of Project 3">
+      </div>
+      <div class="project-content">
+        <h2>Tonic</h2>
+        <div class="project-info">
+          <h3>CANOPY</h3>
+          <p>Back End Dev</p>
+          <p>2015</p>
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li> 
+          <li><a href="#">CSS</a></li> 
+          <li><a href="#">JavaScript</a></li> 
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
+     <!-- 4th project card -->
+    <article class="project-card">
+      <div class="project-img">
+        <img src="assets/Snapshoot-4.svg" alt="Screenshot of Project 4">
+      </div>
+      <div class="project-content">
+        <h2>Multi-Post Stories</h2>
+        <div class="project-info">
+          <h3>CANOPY</h3>
+          <p>Back End Dev</p>
+          <p>2015</p>
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
+  </section>
+   <!-- ABOUT SECTION -->
+   <section id="about-section" class="about-section">
```

Comment:
Please use `Flexbox` to align the content of the "About myself" section as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), 👍🛠